pkg/cmd/pipelineascode/pipelineascode.go

```diff
@@ -60,7 +60,7 @@ func runWrap(p cli.Params, opts *pacpkg.Options) error {
 	}
 
 	err = pacpkg.Run(cs, runInfo)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "403 Resource not accessible by integration") {
```

Comment:
That should only happen on dev test and messing around with INSTALLATION_ID number